src/lib.rs

```diff
@@ -113,6 +130,72 @@ pub fn run_app() {
     let mut last_fps_print = Instant::now();
     let mut frame_counter = 0;
 
+    let image = image::load(
```

Comment:
Would like to keep main clean, so please move all the plane stuff to another file

Reply:
Done, moved it to plane_renderer.rs